pkg/extract/oracle/testdata/golden/modularitylabel-stream-reversed/data/2024/ELSA-2024-4197.json

```diff
@@ -1,8 +1,8 @@
 {
-	"id": "oval:com.oracle.elsa:def:20244197",
+	"id": "ELSA-2024-4197",
 	"advisories": [
 		{
-			"id": "oval:com.oracle.elsa:def:20244197",
+			"id": "ELSA-2024-4197",
 			"title": "\nELSA-2024-4197:  httpd:2.4/httpd security update (MODERATE)\n",
 			"description": "\nhttpd\n[2.4.37-65.0.1]\n- Replace index.html with Oracle's index page oracle_index.html\n\n[2.4.37-65]\n- Resolves: RHEL-31857 - httpd:2.4/httpd: HTTP response\n  splitting (CVE-2023-38709)\n\nmod_http2\nmod_md\n",
```

Comment:
I'm a bit bothered by the newlines at the beginning and end of the title and description. How about deleting them?
```diff
:100644 100644 56c7f0f0 00000000 M	pkg/extract/oracle/oracle.go

diff --git a/pkg/extract/oracle/oracle.go b/pkg/extract/oracle/oracle.go
index 56c7f0f0..41621078 100644
--- a/pkg/extract/oracle/oracle.go
+++ b/pkg/extract/oracle/oracle.go
@@ -182,8 +182,8 @@ func extract(def oracle.Definition, tos tos) (dataTypes.Data, error) {
 		ID: id,
 		Advisories: []advisoryTypes.Advisory{{
 			ID:          id,
-			Title:       def.Metadata.Title,
-			Description: def.Metadata.Description,
+			Title:       strings.TrimSpace(def.Metadata.Title),
+			Description: strings.TrimSpace(def.Metadata.Description),
 			Severity: []severityTypes.Severity{{
 				Type:   severityTypes.SeverityTypeVendor,
 				Source: "linux.oracle.com/security",

```

Reply:
adopted at 7638c15